recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "3.13.2" %}
+{% set version = "3.13.3" %}
```

Comment:
Should we update to `v3.13.4` instead? Especially, because of critical security issues https://docs.python.org/release/3.13.4/whatsnew/changelog.html#security